hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientService.java

```diff
@@ -88,16 +93,27 @@ public SqlClientService(HazelcastClientInstanceImpl client) {
         long resubmissionTimeoutMillis = client.getProperties().getPositiveMillisOrDefault(INVOCATION_TIMEOUT_SECONDS);
         this.resubmissionTimeoutNano = TimeUnit.MILLISECONDS.toNanos(resubmissionTimeoutMillis);
         this.resubmissionRetryPauseMillis = client.getProperties().getPositiveMillisOrDefault(INVOCATION_RETRY_PAUSE_MILLIS);
+        this.isUniSocket = !client.getClientConfig().getNetworkConfig().isSmartRouting();
```

Comment:
It's better to not invent new terminology: 
```suggestion
        this.isSmartRouting = client.getClientConfig().getNetworkConfig().isSmartRouting();
```